src/js/App/Sidenav/NavigationItem.js

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { NavItem } from '@patternfly/react-core/dist/esm/components/Nav';
 
-const basepath = `${document.baseURI}platform/`;
+const basepath = `${document.baseURI}`;
```

Comment:
This is already been worked on in https://github.com/RedHatInsights/insights-chrome/pull/185